src/syntax/test-rules.rkt

```diff
@@ -53,11 +53,13 @@
   (define ctx (context fv repr (map (curry dict-ref itypes) fv)))
 
   (define pre (dict-ref *conditions* name '(TRUE)))
+  (define spec1 (expand-accelerators (prog->spec p1)))
+  (define spec2 (expand-accelerators (prog->spec p2)))
```

Comment:
Ok, this is fine but not ideal—if you write a benchmark with `expm1` in it Herbie might fail to sample it.

I guess this isn't the end of the world and we can merge now and wait for some later time to fix this properly.